src/cascadia/TerminalCore/Terminal.cpp

```diff
@@ -1053,12 +1053,17 @@ void Terminal::_AdjustCursorPosition(const COORD proposedPosition)
 
     // Firing the CursorPositionChanged event is very expensive so we try not to do that when
     // the cursor does not need to be redrawn.
-    if (!cursor.IsDeferDrawing())
+    if (!cursor.IsDeferDrawing() || _trackingCursorMovement)
     {
         _NotifyTerminalCursorPositionChanged();
     }

```

Comment:
This is annoying because `_NotifyTerminalCursorPositionChanged` causes us to call `TSFInputControl().TryRedrawCanvas()`, which is not necessary when we just want to move the cursor light (which is currently the only purpose of `_trackingCursorMovement`). So we _could_ have another type of the-cursor-has-moved event which will lead to _only_ moving the light, but I feel that would be confusing/messy

Reply:
Doesn't `TSFInputControl().TryRedrawCanvas()` do nothing if the `TSFControl` isn't open? Do we even need to filter these events?